src/_scss/pages/agency/statusOfFunds/_visualizationSection.scss

```diff
@@ -112,24 +117,115 @@
         }
     }
 
-    .status-of-funds__emphasis {
-        white-space: nowrap;
+    .status-of-funds__controls-heading-container {
+        display: flex;
+        flex-wrap: wrap;
+        margin: rem(8) 0;
+        @media(min-width: $tablet-screen) {
+            margin: 0;
+        }
+            //@media(min-width: $medium-screen) {
+            //    margin: rem(16) 0 rem(10) rem(32);
+            //}
```

Comment:
could this commented out text be removed?

Reply:
removed